### PR TITLE
Add patch to qemu-kvm to fix CVE-2021-20203

### PR DESCRIPTION
**Pull request by neha170**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 
Add patch to qemu-kvm to fix resolve CVE-2021-20203

###### Change Log  
- Add patch to qemu-kvm to fix resolve CVE-2021-20203

###### Does this affect the toolchain?  
NO

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2021-20203

###### Test Methodology
- Local build

### CODE REVIEW

**Review by henryli001 — APPROVED** [review]

File: SPECS/qemu-kvm/qemu-kvm.spec [hunk 1]
```diff
@@ -45,6 +45,7 @@ Patch26:        CVE-2018-12617.patch
 Patch27:        CVE-2020-25723.patch
 Patch28:        CVE-2020-27821.patch
 Patch29:        CVE-2020-17380.patch
+Patch30:        CVE-2021-20203.patch
```
Comment (hbeberman) [inline SPECS/qemu-kvm/qemu-kvm.spec:48]: This patch isn't getting applied, you'll need to add a ```%patch30 -p1``` line below.

**Comment by hbeberman** [comment]
Please update the release number of the spec and add a changelog entry saying that you've patched the CVE.

**Review by hbeberman — CHANGES_REQUESTED** [review]
Please update the release number of the spec and add a changelog entry saying that you've patched the CVE.

File: SPECS/qemu-kvm/qemu-kvm.spec (hunk 1 above)
Comment (neha170) [reply SPECS/qemu-kvm/qemu-kvm.spec:48]: Done

**Review by hbeberman — APPROVED** [review]

**Pull request merged**